setup.py

```diff
@@ -3,15 +3,31 @@
 
 try:
     import pypandoc
-    long_description = pypandoc.convert('README.md', 'rst')
+
+    long_description = pypandoc.convert("README.md", "rst")
 except (IOError, ImportError):
-    long_description = 'Thumbor redis storage adapters'
+    long_description = "Thumbor redis storage adapters"
 
 
 def read(fname):
     return open(os.path.join(os.path.dirname(__file__), fname)).read()
 
 
+TESTS_REQUIREMENTS = [
+    "black==22.*,>=22.1.0",
+    "preggy==1.*,>=1.4.4",
+    "pyssim==0.*,>=0.4",
+    "pytest==7.*,>=7.0.0",
+    "pytest-cov==3.*,>=3.0.0",
+    "pytest-asyncio==0.*,>=0.18.0",
+]
+
+RUNTIME_REQUIREMENTS = [
+    "redis==4.*,>=4.1.3",
+    "thumbor==7.*,>=7.0.6",
+    "pre-commit==2.*,>=2.17.0",
+]
+
 setup(
     name="tc_redis",
     version="1.0.1",
```

Comment:
The version probably needs to be bumped given this would be a breaking change? Something like `1.1` ?

Reply:
We did a lot of breaking changes, migrating from python 2 to python 3. So, IMHO something like to `2.0.0` sounds better.

Reply:
@masom ping

Reply:
@devppjr The version needs to be bumped, and it seems there's an agreement on `2.0.0` given the Python 3 requirement. Bump it :)

Reply:
@devppjr I've added you as a maintainer of this repo, thanks for this substantial contribution.